### ISSUE TITLE
Regarding the RuntimeError issue.

### ISSUE THREAD

**Issue by GTing31** [qcraftai/pillarnext#23]
Hello, I have set up the environment using a Dockerfile and processed the dataset. Since I have only one GPU, I have modified the run command to
`torchrun --standalone --nnodes=1 --nproc_per_node=1   \
         ./tools/train.py --config-name nusc_det_pp18_aspp_iou_sp \
         data.train_dataset.root_path=/app/datasets/nuScenes/ \
         dataloader.train.batch_size=4 \
         scheduler.max_lr=0.003 \
         trainer.max_epochs=20 \
         hydra.run.dir=outputs/nusc_pillarnextb`

He encountered a network architecture, but after a max: 20 epochs, a RuntimeError occurred: Not compiled with CUDA support.

![Screenshot 2024-07-17 160232](https://github.com/user-attachments/assets/566677f2-4474-4301-898a-c4b832cf8513)

I have tested and successfully enabled CUDA, and it has recognized the GPU.

Could you please let me know if I might have missed any steps? Thank you for your help.

**Comment by GTing31** [comment]
The issue has been resolved. The Dockerfile did not lock the version of torch-scatter, which caused version 2.1.2 to be downloaded. Downgrading to version 2.1.0 resolved the problem.
